packages/feedback-service/README.md

```diff
@@ -0,0 +1,20 @@
+Feedback Microservice
+=================================================
+
+One platform's server-side Feedback GraphQL microservice. This microservice will send data to Feedback-SPA that is completely optional while initializing SPA owner can check/enable feedback option and can provide basic detail like wants to use Jira/Gitlab based on SPA owner this microservice will create issue when someone will give feedback and will fetch all feedback to send to Feedback-SPA listing page.  
```

Comment:
Can you reword this in some simple sentences? Also, make sure whatever is written here is a part of MVP. For eg. Gitlab integration is not a part of MVP, so no need to mention that.

Reply:
Okay.